src/sentry/notifications/notificationcontroller.py

```diff
@@ -441,15 +442,19 @@ def get_subscriptions_status_for_projects(
                     continue
 
                 has_setting = True
-                subscription_status_for_projects[project] = (
+
+                subscription_status_for_projects[project] = SubscriptionStatus(
```

Comment:
@snigdhas can we pass args to it via kwargs to make it more obvious which each argument means?